.github/workflows/php-ci.yml

```diff
@@ -1,6 +1,6 @@
 name: PHP-CI
 
-on: [ push, pull_request ]
```

Comment:
Why?

Reply:
We have double checks on every new commit. We use only `push` in Payments, and all works fine.

Reply:
@natepage any objections?

Reply:
There was a reason why I've added pull_request so let's keep it for now please

Reply:
I have returned this.